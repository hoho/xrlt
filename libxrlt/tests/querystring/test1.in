hello=wo%20r
ld&this=is&this

=IS;very=co%20ol&blah;=he;hi&=ha
