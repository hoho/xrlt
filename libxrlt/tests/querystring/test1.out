<?xml version="1.0"?>
<root><hello>wo rld</hello><this>is</this><this>IS</this><very>co ol</very><blah/>he<hi/>ha</root>
